docs/guides/abci/forum-application/1.abci-intro.md

```diff
@@ -0,0 +1,35 @@
+# Introduction to ABCI++
+
+`ABCI++` is an evolution of ABCI (Application Blockchain Interface) from Cometbft. It acts as an interface between Cometbft (a state machine replication engine) and the actual state machine being replicated (i.e. the application).
+
+![ABCI++](images/ABCI++.jpg)
+
+## ABCI++ Functions
+
+### PrepareProposal
+
+This function prepares a proposed block by processing transactions, performing any necessary validation or filtering, and adding or modifying data as needed.
+
+### ProcessProposal
+
+This function validates and executes the proposed block, applying the necessary changes to the blockchain state based on the transactions included in the block.
+
+### ExtendVote
+
+This function extends the voting period for a proposed block, allowing more time for consensus nodes to vote and reach agreement on the validity of the block.
```

Comment:
Nope. This function extends the votes with data from the voters (validators).

Reply:
modified